PubSub/migrating.md

```diff
@@ -0,0 +1,139 @@
+This guide talks about changes from `google/cloud-pubsub` `v1` to `v2` and how to easily upgrade with some minor changes to your codebase.
+
+## How to upgrade
+
+If your `composer.json` uses (version ranges)[https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators] or exact versions for `google/cloud-pubsub`, then you may need to update those ranges and then run `composer update`.
```

Comment:
I think it would be better to just say something like this:
~~~suggestion
## How to upgrade

Update your `google/cloud` pubsub dependency to `^2.0`:

```
{
    "require": {
        "google/cloud-pubsub": "^2.0"
    }
}
```
~~~